### PR TITLE
Check existence and validity of package.json

### LINKED ISSUE #290: Gracefully handle missing "scripts" block in package.json
Currently, when attempting to run `chromatic` in a project with a package.json that doesn't contain the `"scripts"` property, we get the following error:

```
Cannot read property 'build-storybook' of undefined
```

We should tell the user to add a "scripts" section to their package.json.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1199921881886840/1200015131426373) by [Unito](https://www.unito.io)

### LINKED ISSUE #291: Gracefully handle missing package.json
Currently, when attempting to run `chromatic` in a project without a package.json, we get the following error and the CLI hangs:

```
Unhandled promise rejection: The "path" argument must be one of type string, Buffer, or URL. Received type object
```

We should tell the user they are probably running from the wrong directory.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1199921881886840/1200015179508696) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by ghengeveld**
Fixes #290 and fixes #291

### CODE REVIEW

File: bin/main.js [hunk 1]
```diff
@@ -24,8 +26,18 @@ export async function main(argv) {
   const sessionId = uuid();
   const env = getEnv();
   const log = createLogger(sessionId, env);
+
   const packagePath = await pkgUp(); // the user's own package.json
+  if (!packagePath) {
+    log.error(noPackageJson());
+    process.exit(253);
```
Comment (kylesuss) [inline bin/main.js:33]: What does this number represent (253)?

**Review by kylesuss — APPROVED** [review]
LGTM

File: bin/main.js (hunk 1 above)
Comment (ghengeveld) [reply bin/main.js:33]: Nothing, I just picked one that wasn't used yet.

Comment (coderkevin) [reply bin/main.js:33]: I'm thinking in a future PR maybe we should assign constants to these exit codes, so we can have a list of what they mean and know which ones are used or not.

File: bin/ui/messages/errors/noPackageJson.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+
+import { error } from '../../components/icons';
+
+export default () =>
+  dedent(chalk`
+    ${error} {bold No package.json found}
+    Chromatic only works from inside a JavaScript project.
+    Expecting a package.json somewhere up the directory tree.
```
Comment (coderkevin) [inline bin/ui/messages/errors/noPackageJson.js:10]: Optional: This is a sentence fragment in between two full sentences. May want to make this a full sentence as well.

**Review by coderkevin — APPROVED** [review]
Looks good, just a couple optional comments.

**Pull request merged**